lib/commands/submission.js

```diff
@@ -68,21 +74,31 @@ function exportSubmission(argv, problem, cb) {
     if (e) return cb(e);
     if (submissions.length === 0) return cb('no submissions?');
 
-    // find the latest accepted one
-    var submission = _.find(submissions, function(x) {
-      // TODO: select by lang?
+    // get obj list contain required filetype
+    var submissionInTargetType = _.filter(submissions, function(x) {
+      if (argv.lang === 'all') {
+        return true;
+      }
+      return argv.lang === x.lang;
```

Comment:
Thanks for suggestion! I'll replace them with this one line statement. 
I'm sorry that I might mess up this pull request with too many commits, seems that I made a mistake during rebase. I would like to close this PR and make a new PR with one commit.
Sorry for the inconvenience.